### PR TITLE
update opcua readme

### PR DESCRIPTION
**Pull request by sjwang90**
### Required for all PRs:

- [ ] Signed [CLA](https://influxdata.com/community/cla/).
- [ ] Associated README.md updated.
- [ ] Has appropriate unit tests.

### CODE REVIEW

File: plugins/inputs/opcua/README.md [hunk 1]
```diff
@@ -1,73 +1,73 @@
...
+## certificate = "/etc/telegraf/cert.pem"
 #
-#  # Path to private key.pem. Required for security mode/policy != None. If key path is not supplied, self-signed cert and key will be generated.
-#  # private_key = "/etc/telegraf/key.pem"
+## Path to private key.pem. Required for security mode/policy != None. If key path is not supplied, self-signed cert and key will be generated.
+## private_key = "/etc/telegraf/key.pem"
 #
-#  # To authenticate using a specific ID, select chosen method from 'Certificate' or 'UserName'. Else use 'Anonymous.' Defaults to 'Anonymous' if not provided.
-#  # auth_method = "Anonymous"
+## To authenticate using a specific ID, select chosen method from 'Certificate' or 'UserName'. Else use 'Anonymous.' Defaults to 'Anonymous' if not provided.
```
Comment (noramullen1) [inline plugins/inputs/opcua/README.md:38]: What does "else use" mean here? That you should pick anonymous if you're not using a specific ID to authenticate?

Comment (sjwang90) [reply plugins/inputs/opcua/README.md:38]: yeah so maybe just or else use anonymous if you're not using `auth_method = "Anonymous"` or `auth_method = "Certificate"`

File: plugins/inputs/opcua/README.md [hunk 2]
```diff
@@ -1,73 +1,73 @@
...
-## s=Temperature is indicting that the identifier type is a 'string' and the indentifier value is 'Temperature'
-## This temperature node may have a current value of 79.0, which would possibly make the value a 'float'.
-## To gather data from this node you would need to enter the following line into 'nodes' property above:
-##     {name="SomeLabel", namespace="3", identifier_type="s", identifier="Temperature", data_type="float", description="Some description."},
-
+### Example Node Configuration
+An OPC UA node ID may resemble: "n=3,s=Temperature". In this example:
+- n=3 is indicating the `namespace` is 3
+- s=Temperature is indicting that the `identifier_type` is a string and `identifier` value is 'Temperature'
+- This example temperature node would have a value of 79.0, which would make the `data_type` a 'float'.
```
Comment (noramullen1) [inline plugins/inputs/opcua/README.md:64]: replace "would have" and "would make" with "has" and "makes"

File: plugins/inputs/opcua/README.md [hunk 3]
```diff
@@ -1,73 +1,73 @@
...
+To gather data from this node you would need to enter the following line into the 'nodes' property above:
```
Comment (noramullen1) [inline plugins/inputs/opcua/README.md:65]: To gather data from this node, enter the following....

**Review by reimda — APPROVED** [review]

**Pull request merged**